AEPAssurance/Source/Socket/WebView/WebViewSocket.swift

```diff
@@ -84,6 +84,12 @@ class WebViewSocket: NSObject, SocketConnectable, WKNavigationDelegate, WKScript
             Log.debug(label: AssuranceConstants.LOG_TAG, "Waiting for webView to be loaded open socket connection.")
             return
         }
+        
+        // Sanitize the URL to avoid JS injection attack
+        // BASE Socket URL: "wss://connect%@.griffon.adobe.com/client/v1?sessionId=%@&token=%@&orgId=%@&clientId=%@"
+        if !url.isSafe {
```

Comment:
Sorry I missed reviewing this before it was merged! 
For completeness and to be extra safe, think we should also verify that the URL does not have any additional parameters other than the ones we want.